import/cleaner.php

```diff
@@ -39,50 +39,73 @@ public function __construct() {
 		$this->urlBuilder = new Brizy_Editor_UrlBuilder( $this->project );
 	}
 
-	/**
-	 * @throws Exception
-	 */
-	public function clean() {
-		$this->deleteFiles();
-		$this->cleanTables();
-	}
-
-	private function deleteFiles() {
-
-		$ids = $this->wpdb->get_results( "SELECT p.ID FROM {$this->wpdb->posts} p WHERE p.post_type = 'attachment'", ARRAY_N );
-
-		foreach ( $ids as $id ) {
-			wp_delete_attachment( $id, true );
-		}
-
-		$this->fileSystem->delete( $this->urlBuilder->brizy_upload_path(), true );
-	}
+    /**
+     * @throws Exception
+     */
+    public function clean() {
+        $brzPostTypes = $this->getPostTypes();
+        $placeholders = implode( ', ', array_fill( 0, count( $brzPostTypes ), '%s' ) );
+
+        $this->deletePostRelationships( $brzPostTypes, $placeholders );
+        $this->deletePostsComments( $brzPostTypes, $placeholders );
+        $this->deleteMedia();
+
+        $this->project->setDataAsJson( json_encode( new stdClass() ) )->saveStorage();
+        $this->project->cleanClassCache();
+    }
+
+    private function getPostTypes() {
+        return [
+            Brizy_Admin_Blocks_Main::CP_GLOBAL,
+            Brizy_Admin_Blocks_Main::CP_SAVED,
+            Brizy_Admin_Fonts_Main::CP_FONT,
+            BrizyPro_Admin_Membership_Membership::CP_ROLE,
+            Brizy_Admin_Layouts_Main::CP_LAYOUT,
+            Brizy_Admin_Stories_Main::CP_STORY,
+            Brizy_Admin_Popups_Main::CP_POPUP,
+            Brizy_Admin_Templates::CP_TEMPLATE,
+            Brizy_Admin_FormEntries::CP_FORM_ENTRY,
+
+            // not deleted
+            // Brizy_Editor_Project::BRIZY_PROJECT,
+        ];
+    }
+
+    private function deletePostRelationships( $brzPostTypes, $placeholders ) {
+        $this->wpdb->query(
```

Comment:
Vezi can in WP sunt posturi de tip 'revision' care au ca parent_id postul la care e facuta revizia.

Tu aici stergi tot da imi pare ca reviziile raman. Vezi daca poti sa le stergi si pe ele (si vezi ca ele tot au toate relatiile acestea.)